portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -3928,6 +3928,94 @@ definition {
 			fieldValue = "Checkbox 1");
 	}
 
+	@description = "This is a use case for LPS-125860."
+	@priority = "5"
+	test SubmitFormWithRequiredConfirmationTextField {
+		property test.name.skip.portal.instance = "Forms#SubmitFormWithRequiredConfirmationTextField";
```

Comment:
@brunofariasdeo, please add the following properties:

- property portal.acceptance = "false";
- property portal.upstream = "false";

Reply:
Done!